Create breaking-changes-3.21.adoc (#1007)
gravitee-io/gravitee-docs: pages/am/3.x/breaking-changes/breaking-changes-3.21.adoc

= Breaking changes in 3.21
:page-sidebar: am_3_x_sidebar
:page-permalink: am/current/am_breaking_changes_3.21.html
:page-folder: am/installation-guide
:page-layout: am

NOTE: To take advantage of these new features and incorporate these breaking changes, **use the migration guide available link:{{ '/am/current/am_installguide_migration.html' | relative_url }}[here]**.

== Docker images

> From this version, to be complient with https://www.tenable.com/audits/items/CIS_Docker_v1.3.1_L1_Docker_Linux.audit:bdcea17ac365110218526796ae3095b1[CIS_Docker_v1.3.1_L1] the docker images are now using the user `graviteeio`

Reviewer: ```suggestion
From this version, to be compliant with https://www.tenable.com/audits/items/CIS_Docker_v1.3.1_L1_Docker_Linux.audit:bdcea17ac365110218526796ae3095b1[CIS_Docker_v1.3.1_L1] the docker images now use the `graviteeio` user
```

> This means that if you use the official images and deploy them to Kubernetes, nothing changes.

Reviewer: ```suggestion
This means that if you use the official images and deploy them on your k8s installation, nothing changes.
```

> If you build your own Dockerfile from Gravitee images, you must give the right rights to the modifications you add.

Reviewer: ```suggestion
If you build your own Dockerfile from Gravitee images, you must give the correct rights according to your modifications.
```

> If you deploy in `openshift`, you have to add the following configure:

Reviewer: ```suggestion
If you deploy on `openshift`, you have to add the following configuration:
```
